sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -134,12 +134,12 @@ public KeyVaultKeyStore() {
         jreCertificates = JreCertificates.getInstance();
         LOGGER.log(FINE, String.format("Loaded jre certificates: %s.", jreCertificates.getAliases()));
         wellKnowCertificates = SpecificPathCertificates.getSpecificPathCertificates(wellKnowPath);
-        LOGGER.log(FINE, String.format("Loaded well know certificates: %s.", wellKnowCertificates.getAliases()));
+        LOGGER.log(FINE, String.format("Loaded well known certificates: %s.", wellKnowCertificates.getAliases()));
```

Comment:
well-known?

Reply:
Both is OK.

Reply:
wellKnow**n**Certificates

Reply:
option 1: wellKnownCertificates, customCertificates,keyVaultCertificates, classpathCertificates
option 2: current implementation.

Either is OK.